src/test/kotlin/me/modmuss50/mpp/test/DryTest.kt

```diff
@@ -0,0 +1,160 @@
+package me.modmuss50.mpp.test
+
+import kotlinx.serialization.json.Json
+import org.gradle.testkit.runner.TaskOutcome
+import org.intellij.lang.annotations.Language
+import java.io.File
+import kotlin.test.Ignore
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+/**
+ * Enable this test to execute a dry run, this is more to test the discord integration. Create a options.json with all the tokens
+ */
+@Ignore
+class DryTest : IntegrationTest {
```

Comment:
I dont think DryTest is the correct name here, I wonder if it would be best to just rename this to DiscordIntergrationTest